atef/bin/config.py

```diff
@@ -58,6 +58,13 @@ def build_arg_parser(argparser=None):
     https://doc.qt.io/qt-5/qapplication.html#QApplication
     https://doc.qt.io/qt-5/qguiapplication.html#supported-command-line-options
     """
+    argparser.add_argument(
+        "--cache_size",
```

Comment:
```suggestion
        "--cache-size",
```
My opinion - mixing of `-` and `_` is annoying in CLI args